cmd/grr/workflow.go

```diff
@@ -24,16 +25,14 @@ func getCmd(registry grizzly.Registry) *cli.Command {
 
 func listCmd(registry grizzly.Registry) *cli.Command {
 	cmd := &cli.Command{
-		Use:   "list <jsonnet-file>",
+		Use:   "list [-d <directory> [<jsonnet-file>]",
```

Comment:
```suggestion
		Use:   "list <path>",
```

These are neither optional (`[...]`), nor does `-d` take an argument ;)

Reply:
Yeah, I reworked all of those, and the descriptions which still mention jsonnet